returnn/tf/util/basic.py

```diff
@@ -3237,6 +3237,25 @@ def single_strided_slice(x, axis, begin=None, end=None, step=None):
     return tf.strided_slice(x, begin=begins, end=ends, strides=strides)
 
 
+def pad_replicate(x, axes, padding):
+  """
+  :param tf.Tensor x:
+  :param list[int] axes: 
+  :param list[(int,int)] 
```

Comment:
Return type missing.
`padding` not properly documented.
And remove the trailing space.

In general, always keep the code consistent to the other code. See the [contribution rules](https://github.com/rwth-i6/returnn/blob/master/CONTRIBUTING.md).

```suggestion
  :param tf.Tensor x:
  :param list[int] axes:
  :param list[(int,int)] padding:
  :rtype: tf.Tensor
```